.changeset/serious-geese-admire.md

```diff
@@ -0,0 +1,11 @@
+---
+"@logto/core": patch
+---
+
+fix Microsoft EntraID OIDC SSO connector invalid authorization code response bug
```

Comment:
The root cause is stupid, should we attach Azure Docs link as well?

Reply:
updated